### PR TITLE
Update error message pattern for ALT7 guest

### PR DESCRIPTION
**Pull request by PaulYuuu**
1. update corresponding pattern for ALT7 guest
2. fix "set_vlan_cmd" typo

ID: 1646793

Signed-off-by: Yihuang Yu <yihyu@redhat.com>

### CODE REVIEW

File: generic/tests/cfg/vlan.cfg [hunk 1]
```diff
@@ -14,14 +14,14 @@
...
-    RHEL.7, RHEL.6:
+    RHEL.7, ALT.7, RHEL.6:
         msg_pattern = 'RTNETLINK answers: Numerical result out of range'
     Windows:
         # two windows VM, need more time to boot and login
         login_timeout = 720
         vlan_num = 1
         win_vlan_id = 900
-        set_vlan_cmd = powershell -command "Set-NetAdapter -Name '%s' -VlanID ${win_vlan_id} -Confirm:$false"
+        set_vlan_cmd = powershell -command "Set-NetAdapter -Name '%s' -VlanID '%s' -Confirm:$false"
```
Comment (vivianQizhu) [inline generic/tests/cfg/vlan.cfg]: Is there any difference?

Comment (PaulYuuu) [reply generic/tests/cfg/vlan.cfg]: the python script did not use `win_vlan_id` parameter before.
```
        win_vlan_id = int(params.get("win_vlan_id", 900))
        set_vlan_cmd = params.get("set_vlan_cmd")
        set_vlan_cmd = set_vlan_cmd % nicid
```

Comment (vivianQizhu) [reply generic/tests/cfg/vlan.cfg]: @PaulYuuu Yes, so that should be removed from the script.

Comment (PaulYuuu) [reply generic/tests/cfg/vlan.cfg]: If I remove it from the script, where does it get the value of `win_vlan_id`? Also from `vlan.cfg`?

Comment (vivianQizhu) [reply generic/tests/cfg/vlan.cfg]: @PaulYuuu Yes, the original vlan.cfg `${win_vlan_id}`, that's the grammar of Cartesian configuration

**Comment by vivianQizhu** [comment]
@xiywang Request for your review, thanks.

**Pull request merged**